includes/admin/forms/class-metabox-form-data.php

```diff
@@ -225,6 +225,13 @@ function get_settings() {
 								'placeholder' => esc_attr__( 'Give a Custom Amount', 'give' ),
 							),
 						),
+
+						array(
+							'name'		  => 'donation_options_docs',
+							'type'        => 'docs_link',
+							'url'		  => 'https://givewp.com/documentation/core/give-forms/',
+							'title'		  => "Documentation Options",
```

Comment:
I think `Documentation Options` should be `Donation Options` and also should be translatable:
`__( 'Donation Options', 'give' )`